turbo/generators/templates/sample-package.json.hbs

```diff
@@ -1,5 +1,4 @@
-{
```

Comment:
all the /samples-* files will need to be renamed

Reply:
and the config.ts file as well

Reply:
🤦🏻‍♀️ Thanks for catching. Updated!